src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search-form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.bar {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding: 24px 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 16px;
+  line-height: 21.86px;
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 800;
+}
+
+.small-bar {
+  height: 42px;
+  padding: 10px 33px 11px;
+  font-size: 14px;
+  line-height: 19.12px;
+}
+
+.small-bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.small-bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 800;
+}
```

Comment:
Styles from .bar works here, no need to duplicate it

Reply:
box-shadow is different from the .bar, isn't it? and border-radius too